LenovoLegionToolkit.Lib.Automation/Steps/BrightnessControlAutomationStep.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Threading.Tasks;
+using LenovoLegionToolkit.Lib.Controllers;
+using LenovoLegionToolkit.Lib.System;
+using Newtonsoft.Json;
+
+namespace LenovoLegionToolkit.Lib.Automation.Steps
+{
+    public class DisplayBrightnessAutomationStep : IAutomationStep
+    {
+        private readonly DisplayBrightnessController _controller = IoCContainer.Resolve<DisplayBrightnessController>();
+        public string? Brightness { get; }
```

Comment:
This should rather be an `int`, also preferably non-optional with a default value of let's say 50 (percent).

Reply:
Changed to an `int` and added a default value of 50 when the control is created.